plugins/jabra-gnp/jabra-gnp.quirk

```diff
@@ -0,0 +1,11 @@
+# Jabra Evolve2 75 [runtime]
+[USB\VID_0B0E&PID_24D9]
+Plugin = jabra_gnp
+
+# Jabra Evolve2 75 [runtime]
```

Comment:
Just to be sure -- really the same device in the same state with different PIDs?

Reply:
Yes, we have more than 1 PID for the same device depending on some of it's features.